eng/common/TestResources/New-TestResources.ps1

```diff
@@ -414,6 +412,19 @@ try {
         exit
     }
 
+    $templateFile = ""
+    foreach ($file in $templateFiles) {
+        $templateFileDepth = ($templateFile.OriginalFilePath -split "[\\/]").Length
+        $fileDepth = ($file.OriginalFilePath -split "[\/]").Length
```

Comment:
Should the arguments here be the same, or is it intentional one is `[\\/]` and the other is `[\/]`?

Reply:
Good catch.